Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewControllerTests.swift

```diff
@@ -311,6 +311,54 @@ final class NoShippingPledgeViewControllerTests: TestCase {
       RemoteConfigFeature.pledgeOverTime.rawValue: true
     ]
 
+    orthogonalCombos([Language.en], [Device.phone4_7inch, Device.pad])
+      .forEach { language, device in
+        withEnvironment(
+          apiService: mockService,
+          currentUser: User.template,
+          language: language,
+          remoteConfigClient: mockConfigClient
+        ) {
+          let controller = NoShippingPledgeViewController.instantiate()
+          let data = PledgeViewData(
+            project: project,
+            rewards: [reward],
+            selectedShippingRule: .template,
+            selectedQuantities: [reward.id: 150], // To pass the threshold validation
```

Comment:
I think this threshold only need to be 15 since each reward is $10. The screenshot below is for a reward of $1,500 which is a little confusing.